tests/open-ce-tests.yaml

```diff
@@ -16,6 +16,6 @@ tests:
         cd transformers
         mkdir .cache
         export TRANSFORMERS_CACHE=$(pwd)/.cache
-        pytest --ignore='tests/test_processor_speech_to_text.py' --ignore='tests/test_tokenization_speech_to_text.py' --ignore='tests/test_feature_extraction_speech_to_text.py' -k "not test_batch_encode_dynamic_overflowing and not test_multi_gpu_data_parallel_forward" -v ./tests/
+        pytest --ignore='tests/test_processor_speech_to_text.py' --ignore='tests/test_tokenization_speech_to_text.py' --ignore='tests/test_feature_extraction_speech_to_text.py' -k "not test_batch_encode_dynamic_overflowing and not test_multi_gpu_data_parallel_forward and not test_empty_target_text and not test_max_length_integration and not test_prepare_batch_integration and not test_outputs_can_be_shorter and not test_outputs_not_longer_than_maxlen and not test_max_length and not test_prepare_batch and not test_save_pretrained" -v ./tests/
```

Comment:
Can you add some details here on why we need to disable these extra tests.
thanks!